app/views/users/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Here is a list of the users</h1>
\ No newline at end of file
+<body class="d-flex flex-column align-items-center" >
+  <h1>User's Lists</h1>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user), class:"no-style-link" do %>
+      <%= render 'shared/user', user: user %>
+    <% end %>
```

Comment:
✅ Good job using partials. 👏
- [Optional] Although I would suggest rendering a collection instead of using `@users.each` to render each user. This will make your code lean and easy to read.
> something like 
```ruby 
<%= render partial: 'shared/user', collection: @users%>
```